src/main/webapp/js/pages/planner.js

```diff
@@ -92,6 +92,11 @@ async function savePlan(courseList) {
       method: 'POST',
       body: JSON.stringify(data),
     });
+    document.getElementById('save-button-text').innerText = 'Saved!';
+    document.getElementById('save-plan-prompt')
+        .classList.remove('btn-secondary');
+    document.getElementById('save-plan-prompt').classList.add('btn-primary');
```

Comment:
question - is the user allowed to save again? Should this button be disabled instead?

Reply:
I decided  not to disable it so they could save it again under a different name if they wanted.

Reply:
sounds good.